VK/VK/Model/Models.swift

```diff
@@ -1,4 +1,4 @@
-// Model.swift
+// Models.swift
```

Comment:
```suggestion
// NetworkUnit.swift
```
 Имя файла NetworkUnit поменяй